app/controllers/catalog_controller.rb

```diff
@@ -107,13 +107,18 @@ class CatalogController < ApplicationController
     config.add_index_field 'spatial', field: 'cho_spatial_ssim'
     config.add_index_field 'temporal', field: 'cho_temporal_ssim'
 
+    Settings.acceptable_bcp47_codes.each do |code|
```

Comment:
I could just explicitly add `en` and `ar-Arab` as below for this instead of looping through all of them. 🤷‍♂️